app/app.go

```diff
@@ -781,10 +750,10 @@ func NewCanto(
 
 	maxGasWanted := cast.ToUint64(appOpts.Get(srvflags.EVMMaxTxGasWanted))
 	options := ante.HandlerOptions{
-		AccountKeeper:   app.AccountKeeper,
-		BankKeeper:      app.BankKeeper,
-		EvmKeeper:       app.EvmKeeper,
-		StakingKeeper:   app.StakingKeeper,
+		AccountKeeper: app.AccountKeeper,
+		BankKeeper:    app.BankKeeper,
+		EvmKeeper:     app.EvmKeeper,
+		// StakingKeeper:   app.StakingKeeper,
```

Comment:
Why this is commented? Because there is no StakingKeeper field in HandlerOptions now, so can we delete it?

Reply:
Other things all look good to me, but need to make sure that the new binary with this patch runs well on the testnet.

Reply:
StakingKeeper is necessary only for vesting ante handlers, it's used for delegations regarding vested accounts. 

Currently x/vesting is unused on canto, there should be no accounts affected by removing x/vesting